lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const innerRadiusPercent = props.innerRadiusPercent === 0 ? props.innerRadiusPercent :
+        (props.innerRadiusPercent || 50);
+
+      const strokeWidth = 100 - innerRadiusPercent;
+      const r = innerRadiusPercent + (strokeWidth / 2);
+      const strokeDasharray = [(100 - props.percent), props.percent];
+      return (
+        <svg ref={ref} viewBox="-100 -100 200 200" {...props}>
```

Comment:
Passing `...props` will cause the custom props to get passed to the native element as well, which is why you're getting 
```Warning: React does not recognize the `innerRadiusPercent` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `innerradiuspercent` instead. If you accidentally passed it from a parent component, remove it from the DOM element
```
in the console.  What I'd do is start the function with a line like
```
const { innerRadiusPercent, percent, ...svgAttrs } = props;
```
and then you don't have to use `props.` when looking up the other two props either.

Reply:
> start the function with a line like

Thanks done in https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f